packages/react/src/components/header/Header.stories.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+
+import { Header } from './Header';
+import { HeaderUniversalBar } from '../headerUniversalBar/HeaderUniversalBar';
+import { NavigationLink } from '../navigationLink/NavigationLink';
+import { StoryWIPAlert } from '../../internal/storyWIPAlert/StoryWIPAlert';
+
+export default {
+  component: Header,
+  title: 'Components/Header',
+  parameters: {
+    controls: { expanded: true },
+  },
+  args: {},
+};
+
+export const Example = (args) => (
+  <>
+    <StoryWIPAlert />
+    <Header {...args}>Header</Header>
+  </>
+);
+
+export const WithUniversalBar = (args) => (
+  <>
+    <StoryWIPAlert />
+    <Header {...args}>
+      <HeaderUniversalBar primaryLinkText="Helsingin kaupunki" primaryLinkHref="#">
```

Comment:
Otherwise the one that uses the header component may render the subcomponents in any order they want o___o

Reply:
Well this is a conundrum. Users changing the order of components is a valid point but then it would be a case of "didn't read the memo" and the HDS police would take them to jail (was it Mari who's the copper?). I feel like the user experience is simpler with nested components than with render props but this would actually solve the case of adding UniversalBar's navigation links inside NavigationBar in smaller screens. Also I think that readability gets rougher with render props (worrying about ActionBar already with it's many features). What does @minevala think?

Reply:
I think it would be simpler to render the children, at least for now.
We can then later think about how good the API is when we have more children components like action bar.

Reply:
sure :)